src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -122,7 +122,7 @@ export function QueryPageContainerComponent(): JSX.Element {
                 [keyName]: chooseSelectedFiltersService(selectedOption),
             },
         };
-        const newPath: string = generatePathStringService(newFilter, table);
+        const newPath: string = generatePathStringService(newFilter.selectedFilter, filter.mainFilter, table);

```

Comment:
Reject: mixed parameter source (prone to bugs). Please use: (newFilter.selectedFilter, newFilter.mainFilter)

Reply:
Done. Commit "Remove mixed parameter source & unused openProp"